README.md

```diff
@@ -1,38 +1,68 @@
-[![Build Status][travis-image]][travis-url]
-[![Go Report Card][goreportcard-image]][goreportcard-url]
-[![codecov][codecov-image]][codecov-url]
- 
-## Snaps
-
-Package any app for every Linux desktop, server, cloud or device.
-
-Snaps are faster to install, easier to create, safer to run, and they update
-automatically and transactionally so your app is always fresh and never
-broken. You can bring your own build infrastructure or use ours.
-
-Head over to [snapcraft.io](https://snapcraft.io) to get started.
+[![Snapcraft](https://avatars2.githubusercontent.com/u/19532717?s=200)](https://snapcraft.io)
+
+# Welcome to snapd
+
+This is the code repository for **snapd**, the background service that manages
+and maintains installed snaps. 
+
+Snaps are app packages for desktop, cloud and IoT that update automatically,
+are easy to install, secure, cross-platform and dependency-free. They're being
+used on millions of Linux systems every day.
+
+Alongside its various service and management functions, snapd:
+- provides the _snap_ command that's used to install and remove snaps and
+  interact with the wider snap ecosystem
+- implements the confinement policies that isolate snaps from the base system
+  and from each other
+- marshals the interfaces that allow snaps to access specific system resources
```

Comment:
I looked up _to marshal_ in a dictionary and while it does make sense I think it's somewhat farther away on the simple english spectrum. Perhaps this word could be replaced by _governs_?

Reply:
Yes, I think you're right. Governs is a better/clearer word for this.